tests/test_postgres.py

```diff
@@ -962,14 +863,15 @@ def test_get_remote_status(
             "timestamp": datetime.datetime(2016, 3, 30, 17, 4, 20, 271376),
         }
         current_size_mock.return_value = 497354072
-        has_pgespresso_mock.return_value = True
         server_txt_version_mock.return_value = "9.5.0"
+        # server_txt_version_mock.return_value = PostgreSQL.int_version_to_string_version(PostgreSQL.MINIMAL_VERSION)
```

Comment:
Should this line and L874 also be removed?

Reply:
Done.